tracer/src/Datadog.Trace/Ci/Agent/Payloads/CICodeCoveragePayload.cs

```diff
@@ -17,27 +17,13 @@ internal class CICodeCoveragePayload : MultipartPayload
         public CICodeCoveragePayload(int maxItemsPerPayload = DefaultMaxItemsPerPayload, int maxBytesPerPayload = DefaultMaxBytesPerPayload, IFormatterResolver formatterResolver = null)
             : base(maxItemsPerPayload, maxBytesPerPayload, formatterResolver)
         {
-            var agentlessUrl = CIVisibility.Settings.AgentlessUrl;
-            if (!string.IsNullOrWhiteSpace(agentlessUrl))
-            {
-                var builder = new UriBuilder(agentlessUrl);
-                builder.Path = "api/v2/citestcov";
-                Url = builder.Uri;
-            }
-            else
-            {
-                Url = new UriBuilder(
-                    scheme: "https",
-                    host: "event-platform-intake." + CIVisibility.Settings.Site,
-                    port: 443,
-                    pathValue: "api/v2/citestcov").Uri;
-            }
-
             // We call reset here to add the dummy event
             Reset();
         }
 
-        public override Uri Url { get; }
+        public override string EvpSubdomain => "event-platform-intake";
```

Comment:
I'd rather we expanded the obscure Evp acronym to `EventPlatform`, i.e. `EventPlatformSubdomain`, `EventPlatformPath`, `EventPlatformPayload` etc

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3032/commits/39c6902695e4267810c1ebcf3cc70183756e23e4